### PR TITLE
Support for listing merge request notes

### PR DESCRIPTION
**Pull request by dlukman**
This covers listing merge request notes which was missing before.

### CODE REVIEW

File: lib/gitlab/client/notes.rb [hunk 1]
```diff
@@ -81,6 +95,19 @@ def snippet_note(project, snippet, id)
...
     end
 
+    # Gets a single merge_request note.
+    #
+    # @example
+    #   Gitlab.merge_request_note(5, 11, 3)
+    #
+    # @param [Integer] project The ID of a project.
+    # @param [Integer] merge_request The ID of a merge_request.
+    # @param [Integer] id The ID of an note.
```
Comment (asedge) [inline lib/gitlab/client/notes.rb]: `a note` not `an note`

**Comment by asedge** [comment]
Looks pretty good other than the minor grammatical error in the documentation.  Please fix that and squash commits.

**Comment by dlukman** [comment]
Thanks, I fixed the typo.

**Comment by asedge** [comment]
Thanks for the contribution!

**Pull request merged**